PC/msvcrtmodule.c

```diff
@@ -664,10 +648,29 @@ PyInit_msvcrt(void)
                                                   _VC_CRT_BUILD_VERSION,
                                                   _VC_CRT_RBUILD_VERSION);
     st = PyModule_AddObject(m, "CRT_ASSEMBLY_VERSION", version);
-    if (st < 0) return NULL;
+    if (st < 0) return -1;
```

Comment:
We can apply PEP-7 to the code we touch; it is nice to leave the code in a better state than what we found it.

```suggestion
    if (st < 0) {
        return -1;
    }
```

Reply:
You also need to change the three `return NULL`s above.

Reply:
Thanks for review, their macro blocks were folded by the IDE so I missed them.

Reply:
Yeah, I figured that :)